app/assets/v2/js/pages/bulk_payout.js

```diff
@@ -112,7 +112,7 @@ $(document).ready(function($) {
       // get form data
       var email = '';
       var github_url = $('#issueURL').val();
-      var from_name = '';
+      var from_name = document.contxt['github_handle'];
```

Comment:
`contxt` vs `context` ?

Reply:
yea; thats just how the app is rn.  we could update it as part of the PR if ppl feel strongly about it